.github/workflows/go-scheduled.yml

```diff
@@ -0,0 +1,43 @@
+name: Daily scheduled extended Go tests
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
😍

Reply:
Note that this is midnight UTC, is that the time we want?
(I could see it being an issue because it's "only" 5pm PST)

Reply:
Changing it to 6am UTC which is roughly start of shift for European timezone.